src/client/src/misc/Color.cpp

```diff
@@ -0,0 +1,8 @@
+#include <client/misc/Color.hpp>
+
+namespace client {
+
+	void init_palettes()
```

Comment:
Does this have a purpose?

Reply:
This is for the colour redesign - strictly speaking it shouldn't really be in this PR but it's basically dead code so won't actually affect anything.